include/zenoh/api/interop.hxx

```diff
@@ -40,45 +40,45 @@ inline constexpr bool is_loan_mut_available_v = is_loan_mut_available<T>::value;
 
 namespace zenoh::interop {
 
-/// @brief Get C representation of trivially copyable Zenoh-c++ object.
+/// @brief Get C representation of trivially copyable zenoh-c++ object.
 template <class CopyableType>
 CopyableType* as_copyable_c_ptr(Copyable<CopyableType>& cpp_obj) {
     return reinterpret_cast<CopyableType*>(&cpp_obj);
 }
 
-/// @brief Get C representation of trivially copyable Zenoh-c++ object.
+/// @brief Get C representation of trivially copyable zenoh-c++ object.
 template <class CopyableType>
 const CopyableType* as_copyable_c_ptr(const Copyable<CopyableType>& cpp_obj) {
     return reinterpret_cast<const CopyableType*>(&cpp_obj);
 }
 
-/// @brief Get C representation of std::optional of trivially copyable Zenoh-c++ object.
-/// @return Pointer to C representation of trivially copyable Zenoh-c++ object, or NULL if cpp_obj is empty.
+/// @brief Get C representation of std::optional of trivially copyable zenoh-c++ object.
+/// @return Pointer to C representation of trivially copyable zenoh-c++ object, or NULL if cpp_obj is empty.
```

Comment:
may be we should use library name "zenoh-cpp" here ? It is eihter "zenoh C++ binding/library" or "zenoh-cpp", but name "zenoh-c++" looks wrong for me

Reply:
this more refers to c++/c language constructs, rather than libraries themselves. Maybe we can just remove a dash in attempt to try to avoid confusion.

Reply:
fixed C to zenoh-c and C++/zenoh-c++ to zenoh-cpp